### PR TITLE
Enabling 2 Windows E2E tests

### PR DESCRIPTION
**Pull request by PatrickLang**
**What this PR does / why we need it**:
- Adds timestamps to deployments commands in e2e tests. Some Windows deployments take a long time, so this will be helpful in tuning timeout durations
- Reenables 2 Windows tests commented out due to flakes
- Skip a portion of test that's failing to a known bug with DNS setup. Hopefully this will fix the test flake. If it does, I propose moving it to a separate test case focused on outbound network connectivity from Windows pods since that has been flaky over the last few weeks.

I'm relying on the automated tests for this, so there may be another commit or two added before it's ready to merge.

**Special notes for your reviewer**:

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

### CODE REVIEW

**Comment by PatrickLang** [comment]
If it passes this could replace #3241

**Comment by PatrickLang** [comment]
I'm kicking failed tests through CircleCI UI. k8s-linux-default-e2e passed now

File: test/e2e/kubernetes/util/util.go [hunk 1]
```diff
@@ -2,11 +2,24 @@ package util
...
+	"time"
 )
 
 // PrintCommand prints a command string
 func PrintCommand(cmd *exec.Cmd) {
 	fmt.Printf("\n$ %s\n", strings.Join(cmd.Args, " "))
 }
+
+// RunAndLogCommand logs the command with a timestamp when it's run, and duration at and
+func RunAndLogCommand(cmd *exec.Cmd) ([]byte, error) {
```
Comment (PatrickLang) [inline test/e2e/kubernetes/util/util.go]: @jackfrancis @CecileRobertMichon are you ok with this tiny refactor as part of this PR? If not I can split it into a separate PR

Comment (PatrickLang) [reply test/e2e/kubernetes/util/util.go]: before
```
STEP: Creating a nginx deployment

$ kubectl run ingress-nginx-kubernetes-eastus-5726-17635 -n default --image library/nginx:latest --overrides { "apiVersion": "extensions/v1beta1", "spec":{"template":{"spec": {"nodeSelector":{"beta.kubernetes.io/os":"linux"}}}}} --labels=app=ingress-nginx
```


after
```
STEP: Creating a nginx deployment
2018/06/22 16:38:14 $ kubectl run nginx-kubernetes-eastus-67793-62845 -n default --image library/nginx:latest --overrides { "apiVersion": "extensions/v1beta1", "spec":{"template":{"spec": {"nodeSelector":{"beta.kubernetes.io/os":"linux"}}}}}
2018/06/22 16:38:14 #### $ kubectl run nginx-kubernetes-eastus-67793-62845 -n default --image library/nginx:latest --overrides { "apiVersion": "extensions/v1beta1", "spec":{"template":{"spec": {"nodeSelector":{"beta.kubernetes.io/os":"linux"}}}}} completed in 286.941082ms
```

Comment (CecileRobertMichon) [reply test/e2e/kubernetes/util/util.go]: Fine with me but I think there's a typo at the end of the comment

File: test/e2e/kubernetes/util/util.go [hunk 2]
```diff
@@ -2,11 +2,24 @@ package util
...
 	"strings"
+	"time"
 )
 
 // PrintCommand prints a command string
 func PrintCommand(cmd *exec.Cmd) {
 	fmt.Printf("\n$ %s\n", strings.Join(cmd.Args, " "))
 }
+
+// RunAndLogCommand logs the command with a timestamp when it's run, and the duration at end
```
Comment (PatrickLang) [inline test/e2e/kubernetes/util/util.go]: Thanks @CecileRobertMichon - fixed typo

**Pull request merged**